internal/policies/mount/mount_test.go

```diff
@@ -116,7 +121,10 @@ func TestApplyPolicy(t *testing.T) {
 		"system, successfully apply policy filtering out unsupported keys": {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "system-mounts"}, isComputer: true},
 
 		// Special cases.
-		"system, successfully apply policy with kerberos tagged values": {entries: []string{"entry with kerberos auth tags"}, isComputer: true},
+		"system, successfully apply policy with kerberos tagged values":                         {entries: []string{"entry with kerberos auth tags"}, isComputer: true},
+		"system, successfully apply policy prioritizing the first value found, despite the tag": {entries: []string{"entry with same values tagged and untagged"}, isComputer: true},
+		"system, only emit a warning when starting new units and systemctl fails":               {isComputer: true, firstSystemCtlFailingArgs: []string{"start"}},
```

Comment:
TODO: add a new test for stop too, when disable --now is split (see https://github.com/ubuntu/adsys/pull/522#discussion_r1061546752).